scholarly/_scholarly.py

```diff
@@ -23,12 +23,12 @@ def set_retries(self, num_retries: int):
 
         return self.__nav._set_retries(num_retries)
 
-    def use_proxy(self, http: str, https: str):
+    def use_proxy(self, http: str, https: str = None):
         """Setups a proxy without refreshing capabilities.
 
         :param http: the http proxy address
         :type http: str
-        :param https: the https proxy address
+        :param https: the https proxy (default to the same as http)
```

Comment:
Is this really true? the code presented in this commit just sets it to none and makes it optional

Reply:
Yes, see cddbbcd3adfa06df8a35eff24357bc99f206409b, which is the "previous change" I mentioned in the commit message. I could make it more explicit if you like.